### ISSUE TITLE
Need role for column break

### ISSUE THREAD

**Issue by aleventhal** [w3c/dpub-aria#11]
Moved from https://github.com/w3c/dpub/issues/25

Without this semantic, , screen readers cannot provide their users with settings to turn on/off the visibility of  column break, which the user may wish to examine for authoring purposes.

**Comment by TzviyaSiegman** [comment]
after discussion with @aleventhal we concluded that this is more stylistic than semantic.
